docs/_templates/breadcrumbs.html

```diff
@@ -0,0 +1,4 @@
+{%- extends "!breadcrumbs.html" %}
+
+{% block breadcrumbs_aside %}
+{% endblock %}
```

Comment:
I see this is a work in progress, but: what would this do?

Reply:
This part is complete.  This bit of code removes the view page source code link from the header.

I basically copied the [`breadcrumbs_aside` block](https://github.com/readthedocs/sphinx_rtd_theme/blob/1d856256676d2b801a12c30e4a849c5abc466da9/sphinx_rtd_theme/breadcrumbs.html#L39-L70) from the [`breadcrumbs.html` template](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/breadcrumbs.html) and moved it into the [our `footer.html`](https://github.com/rocco8773/PlasmaPy/blob/ec0e100da6de1be968466ff3f242d6b5948c065f/docs/_templates/footer.html#L23-L44).  Then this bit of code wipes out the `breadcrumbs_aside` block so the link does not appear twice.